src/main/java/lotto/service/LottoGame.java

```diff
@@ -0,0 +1,68 @@
+package lotto.service;
+
+import lotto.domain.AutoLotto;
+import lotto.domain.Lotto;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class LottoGame {
+    private static final List<Integer> lottoNumbers = new ArrayList<>();
+    public static final int LOTTO_PRICE = 1000;
+    public static final int LOTTO_NUMBER_COUNT = 6;
+
+    private LottoGame() {
+        IntStream.rangeClosed(1, 45)
+                .forEach(lottoNumbers::add);
+    }
+
+    private static LottoGame INSTANCE = null;
+
+    public static LottoGame getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new LottoGame();
+        }
+        return INSTANCE;
+    }
+
+
+    public int getLottoCount(int priceTotal) {
+        if (priceTotal < 1000) {
```

Comment:
```suggestion
        if (priceTotal < LOTTO_PRICE) {
```
선언해둔 LOTTO_PRICE를 사용하는 것이 맞겠네요 😄

Reply:
상수처리 하면서 수정했습니다!